README.md

```diff
@@ -1,7 +1,43 @@
+# App Configuration Sync action
+
+This action syncs configuration files in the repository to an App Configuration instance. This enables scenarios where the App Configuration instance is automatically updated when changes are made through GitHub workflows.
+
+JSON, YAML, and .properties files are supported. For the full list of action inputs, see [Inputs](./action.yml).

```

Comment:
It might be helpful to end this section with the following.


To start using this GitHub action go to your repository and click the "Actions" tab. This GitHub action will be available from the market place under the name "Azure App Configuration Sync". See the usage section below for an example of how to set up the action yml file. For a more in-depth view of GitHub workflows and actions click [here](https://help.github.com/en/actions/automating-your-workflow-with-github-actions).